addon/components/paper-menu-content-inner.js

```diff
@@ -1,13 +1,24 @@
 import Ember from 'ember';
 import ParentMixin from 'ember-paper/mixins/parent-mixin';
-const { Component, inject } = Ember;
+const { Component, inject, computed, run: { later } } = Ember;
```

Comment:
generally, we are not destructuring more than one level to keep some context. So here, we would use `run.later`.